com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -3108,18 +3111,15 @@ RendererListDesc PrepareForwardTransparentRendererList(CullingResults cullResult
             RendererList                rendererList,
             RenderTargetIdentifier[]    renderTarget,
             RTHandle                    depthBuffer,
-            ComputeBuffer               lightListBuffer,
+            ComputeBuffer               fineTileBuffer,
+            ComputeBuffer               zBinBuffer,
             bool                        opaque,
             ScriptableRenderContext     renderContext,
             CommandBuffer               cmd)
         {
-            // Note: SHADOWS_SHADOWMASK keyword is enabled in HDRenderPipeline.cs ConfigureForShadowMask
-            bool useFptl = opaque && frameSettings.IsEnabled(FrameSettingsField.FPTLForForwardOpaque);
-
-            // say that we want to use tile/cluster light loop
-            CoreUtils.SetKeyword(cmd, "USE_FPTL_LIGHTLIST", useFptl);
-            CoreUtils.SetKeyword(cmd, "USE_CLUSTERED_LIGHTLIST", !useFptl);
-            cmd.SetGlobalBuffer(HDShaderIDs.g_vLightListGlobal, lightListBuffer);
+            // TODO: we have so many SetGlobal calls like this that it makes more sense to just do it once...
```

Comment:
you can remove the comment, we have no choice anyway as we need to bind for forward.